src/model_theory/definability.lean

```diff
@@ -156,95 +153,95 @@ end
 variables (L) {M} (A)
 
 /-- A 1-dimensional version of `definable`, for `set M`. -/
-def definable₁ (s : set M) : Prop := L.definable A { x : fin 1 → M | x 0 ∈ s }
+def definable₁ (s : set M) : Prop := A.definable L { x : fin 1 → M | x 0 ∈ s }
 
 /-- A 2-dimensional version of `definable`, for `set (M × M)`. -/
-def definable₂ (s : set (M × M)) : Prop := L.definable A { x : fin 2 → M | (x 0, x 1) ∈ s }
+def definable₂ (s : set (M × M)) : Prop := A.definable L { x : fin 2 → M | (x 0, x 1) ∈ s }
 
-variables (α)
+variable (α)
 
 /-- Definable sets are subsets of finite Cartesian products of a structure such that membership is
   given by a first-order formula. -/
-def definable_set := subtype (λ s : set (α → M), L.definable A s)
+def definable_set := subtype (λ s : set (α → M), A.definable L s)
```

Comment:
```suggestion
def definable_set := { s : set (α → M) // A.definable L s }
```
Maybe this should also be *outside* the `set` namespace?

Reply:
I've reverted this one to `first_order.language.definable_set`.